### DISCUSSION TITLE
[1.5.1] 2022-03-29

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#241] [General]
# Wired
* [PCE] Rework 2 btns to ignore OE line and handle OE via interrupt for MT & 6 btns (Fixes #192 #212 )

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.5.1'>[1.5.1] 2022-03-29</a>.</em>

**Comment by pmgducati** [comment]
Are there any changes needed to the connections between the ESP and PCE?

**Reply by darthcloud** [reply]
No

**Comment by Tredes** [comment]
How could I solder the blueretro module inside the MULTITAP PS1 to control the 4 players? thanks

**Reply by darthcloud** [reply]
You dont do that.

You just plug as usual on one port and enable multitap mode in config.

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-System-Specific-User-Manual#psxps2
